api/envoy/extensions/filters/network/tcp_proxy/v3/tcp_proxy.proto

```diff
@@ -184,6 +184,8 @@ message TcpProxy {
   // is defined as the period in which there are no bytes sent or received on either
   // the upstream or downstream connection. If not set, the default idle timeout is 1 hour. If set
   // to 0s, the timeout will be disabled.
+  // It is possible to dynamically override this configuration by setting a per-connection filter
+  // state object for the key ``envoy.tcp_proxy.per_connection_idle_timeout_ms``.
```

Comment:
Can you provide a link to this configuration setting?

Reply:
Hi, what do you mean by configuration setting?

Reply:
It wasn't clear to me reading this where the per-connection filter state object is set, so I was wondering if you could provide a pointer to it in the comment.  I realized just now that the comment is very similar to the one in the same file here: https://github.com/envoyproxy/envoy/blob/3a43d39256a85eae9d90368161e0a8fbbb5c1952/api/envoy/extensions/filters/network/tcp_proxy/v3/tcp_proxy.proto#L220

It's just that to me, as a reader, it's not clear where I would set that dynamic override at first glance, which is why I was suggesting a reference to where the per-connection filter state object is set.

Thanks!

Reply:
There isn't a specific object for it, because you'd need to set the filter state value with a ``UInt64Accessor`` object which already exists. The filter state object would need to be set by an external downstream dev in some extension code, to allow this override. By default, there'll be no override and this pattern is similar to the other filter state overrides as described in the "well known filter state" docs (modified in this PR).

Reply:
OK, reading through the tests, I understand now what you're referring to